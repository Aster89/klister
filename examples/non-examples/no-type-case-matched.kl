#lang "prelude.kl"

(import (shift "prelude.kl" 1))

(define-macros
  ([m1
    (lambda (stx)
      (>>= (which-problem)
        (lambda (problem)
          (case problem
            [(expression tp)
             (type-case tp
               [(Integer)
                (pure '42)])]))))]))

(example (the String m1))
