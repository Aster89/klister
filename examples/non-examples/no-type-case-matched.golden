Error at phase p0:
    No case matched at no-type-case-matched.kl:12.14-14.29: String
